drivers/interrupt_controller/Kconfig.intel_vtd

```diff
@@ -15,9 +15,20 @@ menuconfig INTEL_VTD_ICTL
 
 if INTEL_VTD_ICTL
 
+config INTEL_VTD_ICTL_XAPIC_PASSTHROUGH
+	bool "XAPIC mode pass-through"
+	depends on !X2APIC
+	help
+	  If XAPIC mode is enabled, it will avoid remapping all interrupts.
+
+config INTEL_VTD_ICTL_NO_SRC_ID_CHECK
+	bool "Never check the source id"
+	help
+	  This disables the source id check in IRTE.
```

Comment:
help text should omit leading with "this", "this option", "enabling this", etc. in the sentence.

So:

Disable the source id check in IRTE.

Reply:
Options should enable things whenever possible.

So it should be

bool "check the source id"
default y
help
Check the source id in IRTE.

Reply:
still not resolved